lib/src/lifecycle_module.dart

```diff
@@ -77,12 +77,12 @@ abstract class LifecycleModule {
   Future loadChildModule(LifecycleModule newModule) async {
     newModule.didLoad.listen((_) {
       _childModules.add(newModule);
-      _didLoadChildModuleController.add(newModule);
     });
     newModule.willUnload.listen((_) {
       _childModules.remove(newModule);
     });
     await newModule.load();
+    await _didLoadChildModuleController.add(newModule);
```

Comment:
I think the previous code was more correct/better composed, with the event dispatch inside the `newModule.didLoad` handler. But again, I am very curious why Dart 1.13 necessitated this change.